server/views.py

```diff
@@ -397,7 +408,8 @@ def export_users_xls(request):
     # Sheet body, remaining rows
     font_style = xlwt.XFStyle()
 
-    rows = Volunteer.objects.all().values_list('full_name', 'first_name', 'last_name', 'tz_number', 'volunteer_type', 'age',
+    rows = Volunteer.objects.all().values_list('full_name', 'first_name', 'last_name', 'tz_number', 'volunteer_type',
+                                               'age',
                                                'phone_number', 'areas', 'city', 'email', 'available_saturday',
                                                'guiding', 'notes')
```

Comment:
but why? just use use list(Volunterr.objects.all())

Reply:
Is not really part of my changes, this line changed because of pycharm auto PEP8.